src/main/java/build/buildfarm/common/config/GrpcMetrics.java

```diff
@@ -10,6 +10,7 @@ public class GrpcMetrics {
   private boolean enabled = false;
   private boolean provideLatencyHistograms = false;
   private double[] latencyBuckets;
+  private String labelsToReport = "";
```

Comment:
Make this an array, no need to split based on undocumented comma separator.

Reply:
Switched it to a List, since that is what `grpcConfig.withLabelHeaders` takes. Let me know if an Array field with a Array->List conversion at the time of calling is preferred.

The reason this variable was a string previously was due to the variable being loaded through an environmental variable in production, which can only be a string without additional conversion. Where can I ensure that `BuildfarmConfigs` contains the correct format of `labelsToReport`?